docs/cli/build/in-obs.md

```diff
@@ -0,0 +1,51 @@
+---
+title: in-obs
+date: 2024-06-27
+sidebar_position: 5
+---
+
+# `packit build in-obs`
+
+Submit a build of the present content in the upstream or local repository to [OBS](https://build.opensuse.org/).
+
+## Requirements
+
+* SPEC file for the project placed in the repository.
+* Packit config file placed in the repository.
+
+
+
+## Tutorial
+
+1. [Place a config file for packit in the root of your upstream repository.](/docs/configuration/)
+
+2. The command below creates a tarball (archive) of your repository and commits it  along with the spec file to a package with a name corresponding to your upstream project on [OBS](https://build.opensuse.org/). The project defaults to home:$username:packit. Running packit build in-obs a second time creates a new commit in that package. 
+
+    ```
+    $ cd my/ustream/project/
+    $ packit build in-obs --project <project_name>
+    ```
+
+## Help
```

Comment:
I am not sure if you know this, but this is being regenerated automatically from the real help message so you don't need to take care of updating.. Also, what you put into the help message can be edited in one place (~ `packit` project) and is then visible both in the help message and on docs...

Reply:
The script is [here](https://github.com/packit/packit.dev/blob/main/.github/workflows/regenerate-cli-help-messages.yml).

Reply:
Okay so the help section is updated automatically

Reply:
can we leave the other sections in?